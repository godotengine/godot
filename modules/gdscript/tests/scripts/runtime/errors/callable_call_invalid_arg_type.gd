#debug-only
func test():
	print(load.bind([]).call())
