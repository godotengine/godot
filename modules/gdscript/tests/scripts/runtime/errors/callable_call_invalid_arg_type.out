GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR at runtime/errors/callable_call_invalid_arg_type.gd:3 on test(): Invalid type in function '@GDScript::load (Callable)'. Cannot convert argument 1 from Array to String.
